src/App.js

```diff
@@ -5,25 +5,20 @@ import './App.scss';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import { Stats } from './components/Stats';
+import { PostList } from './components/PostList';
 
 const App = () => (
   <div className="App">
     <h1>Static list of posts</h1>
 
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
+    <Stats
+      posts={posts.length}
+      comments={comments.length}
+      users={users.length}
+    />
 
-    <p>
-      <span>comments: </span>
-      {comments.length}
-    </p>
-
-    <p>
-      <span>Users: </span>
-      {users.length}
-    </p>
+    <PostList posts={posts} />
```

Comment:
pass here preparedPost array, which already contains all the necessary information on each post (user and comments)